cli/ballerina-cli/src/main/resources/cli-help/ballerina-shell.help

```diff
@@ -0,0 +1,33 @@
+NAME
+       ballerina-shell - Run ballerina interactive REPL
+
+SYNOPSIS
+       bal shell [-d|--debug] [-f|--force-dumb]  [-t|--time-out]
+
+
+DESCRIPTION
+       Shell runs a REPL instance of ballerina to enable users to run small
+       snippets to code to check their outputs. Please not that this is an
+       experimental feature.
+
+       Debug messages can be enabled via -d parameter.
+
+OPTIONS
+       -d
+           Enable debug mode from the beginning. Not enabled by default.
+       -f
+           Force dumb terminal mode. Not dumb by default.
+       -t <time-out-duration>
```

Comment:
```suggestion
       -t, --time-out
```

Reply:
I was wrong here in removing `<time-out-duration>`.

Shall we make this

```
       -t <time-out-duration>, --time-out <time-out-duration>
```

Reply:
OK